src/components/Category/Category.tsx

```diff
@@ -1,46 +1,24 @@
 import { useTranslations } from "next-intl";
+import SearchBar from "../ui-custom/SearchBar";
+import CategoryList from "./CategoryList";
 
-import { Button } from "@/components/ui/button";
-
-import { Link } from "@/i18n/routing";
-
-// TODO - get categories from database, think about categories we want
-const categories = [
-  { name: "DESSERTS 🍰", href: "/desserts" },
-  { name: "MAIN-DISHES 🍽️", href: "/main-dishes" },
-  { name: "APPETIZERS 🧀", href: "/appetizers" },
-  { name: "SALADS 🥗", href: "/salads" },
-  { name: "SOUPS 🍲", href: "/soups" },
-  { name: "BREAKFAST 🍳", href: "/breakfast" },
-  { name: "BEVERAGES 🧃", href: "/beverages" },
-  { name: "SNACKS 🍿", href: "/snacks" },
-  { name: "SAUCES 🍯", href: "/sauces" },
-  { name: "BREADS 🥖", href: "/breads" },
-  { name: "SEAFOOD 🦞", href: "/seafood" },
-  { name: "VEGETARIAN 🥦", href: "/vegetarian" },
-  { name: "GRILLED 🍖", href: "/grilled" },
-  { name: "HOLIDAY-SPECIALS 🎄", href: "/holiday-specials" },
-  { name: "KIDS-MENU 🍔", href: "/kids-menu" },
-];
-
-const Category = () => {
+const Category = ({
+  searchParams,
+}: {
+  searchParams?: {
```

Comment:
could we just pass here query? without `searchParams` object?

Reply:
yes